### ISSUE TITLE
Mailing through mailjet API fails silently when quota is exceeded

### ISSUE THREAD

**Issue by hughbris** [getgrav/grav-plugin-email#162]
I have the free account with Mailjet. I just noticed mail is not sending. I checked and my quota has been exceeded.

There is nothing in Grav logs and no failure in the form submission UI. So it's a silent failure, which is kind of worst case for a contact form.

I think the problem may lie with the API not returning a failed status. Is this a known problem with other APIs supported by this plugin?

**Comment by rhukster** [comment]
Email 4.0 will have an accompanying Mailjet plugin that lets you use the API interface. This provides loads more tracking and debug information than SMTP. I suggest you test this (https://getgrav.org/blog/new-email-plugin)

**Comment by hughbris** [comment]
Looks like a very promising development, thank you!

Timely too as I just spent the last two days mostly troubleshooting what turned out to be limit-based errors (not silent) with other providers I used after this issue. Then I spent some time trying to "load balance" sites and the free providers I use. Fragile. Which makes me think that fallback providers might be a worthwhile feature in the new version. Hmm.
